libs/single-spa-angular/internals/src/types.ts

```diff
@@ -4,7 +4,7 @@ import { NgModuleRef } from '@angular/core';
 export type DomElementGetter = () => HTMLElement;
 
 export interface BaseSingleSpaAngularOptions {
-  template: string;
+  template: string | ((props: AppProps) => string);
```

Comment:
Allowing template to be a function could be a good change, but doesn't seem related to mounting multiple instances of the same parcel config. Could you separate it into a new PR?

Reply:
We need the reference to the parcel name in order to read the properties specific to the targeted web component instance (we could, of course generate a property containing the parcel name in some way and insert it in the tag). The problem on the angular side, is that we insert the element over innerHtml and we don't have the correspondent properties directly connected to the angular element, as it is with react for example. We need to recreate the connection between the custom properties and the angular element, which is practically a singleton.

Reply:
I don't think I understand - why would the template need to be different for each instance of a parcel? Doesn't each instance of a parcel create a new instance of the angular element, with the single-spa props already passed to the angular element? I didn't implement the angular elements portion of single-spa-angular, so might be missing something about how it works.